src/domain/community/ai-persona/ai.persona.entity.ts

```diff
@@ -3,28 +3,23 @@ import { IAiPersona } from './ai.persona.interface';
 import { AuthorizableEntity } from '@domain/common/entity/authorizable-entity';
 import { AiPersonaInteractionMode } from '@common/enums/ai.persona.interaction.mode';
 import { AiPersonaDataAccessMode } from '@common/enums/ai.persona.data.access.mode';
+import { MINI_TEXT_LENGTH, SMALL_TEXT_LENGTH } from '@common/constants';
 
 @Entity()
 export class AiPersona extends AuthorizableEntity implements IAiPersona {
   // No direct link; this is a generic identifier
-  @Column('varchar', { nullable: false, length: 128 })
+  @Column('varchar', { nullable: false, length: SMALL_TEXT_LENGTH })
   aiPersonaServiceID!: string;
 
   @Column('text', { nullable: true })
-  description = '';
+  description?: string;
 
-  @Column('varchar', {
-    length: 255,
-    default: AiPersonaDataAccessMode.SPACE_PROFILE_AND_CONTENTS,
-  })
+  @Column('varchar', { length: MINI_TEXT_LENGTH, nullable: false })
   dataAccessMode!: AiPersonaDataAccessMode;
 
-  @Column('simple-array', {
-    nullable: false,
-    default: [AiPersonaInteractionMode.DISCUSSION_TAGGING],
-  })
+  @Column('simple-array', { nullable: false })
   interactionModes!: AiPersonaInteractionMode[];
 
   @Column('text', { nullable: true })
-  bodyOfKnowledge = '';
+  bodyOfKnowledge?: string;
```

Comment:
I think this is an identifier, so should be varchar + probably small text length. Please check the usage. We may need to discuss this one.

Reply:
This is `text`, limited to `65568` in the API layer.
There is another `bodyOfKnowledgeID` which is the identifier.

Reply:
Ok, so it is actually the bodyOfKnowledge description. Then ok as is you have it above.